hugo/layouts/papers/single.html

```diff
@@ -98,8 +318,39 @@ <h2 id="title">
   {{ end }}
   </p>
   </div>
-  <hr />
+  <div class="modal fade" id="metadataModal" tabindex="-1" aria-labelledby="metadataModalLabel" aria-hidden="true">
+    <div class="modal-dialog modal-lg">
+      <div class="modal-content">
+        <div class="modal-header">
+          <h5 class="modal-title">Correct Metadata for <span id="paperIdSpan"></span></h5>
+          <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+        </div>
+        <div class="modal-body">
+          <form id="metadataForm">
+            <div class="mb-3">
+              <label for="paperTitle" class="form-label">Title</label>
+              <small id="paperTitleHelp" class="form-text text-muted">Adjust the title. Retain tags such as &lt;fixed-case&gt;.</small>
+              <input type="text" class="form-control" id="paperTitle">
+            </div>
+            <label class="form-label">Authors</label>
+            <small id="authorTitleHelp" class="form-text text-muted">Adjust author names, affiliations, and list order to match the PDF.</small>
+            <div id="authorsContainer" ondrop="dropAuthor(event)" ondragover="allowDrop(event)"></div>
```

Comment:
Add `class="overflow-x: scroll;"` to prevent issues with narrower screens.

Reply:
Additionally, I think we can make the name fields shorter on smaller screens, using responsive Bootstrap classes. I can make a suggestion tomorrow.